frontend/ui/network/manager.lua

```diff
@@ -117,6 +119,21 @@ function NetworkMgr:getProxyMenuTable()
     }
 end
 
+function NetworkMgr:getRestoreMenuTable()
+    return {
+        text = _("Automatically restore Wi-Fi connection after resume"),
+        checked_func = function() return G_reader_settings:readSetting("auto_restore_wifi") end,
+        enabled_func = function() return Device:isKobo() end,
+        callback = function(menu)
+            if G_reader_settings:readSetting("auto_restore_wifi") then
+                G_reader_settings:delSetting("auto_restore_wifi")
+            else
+                G_reader_settings:saveSetting("auto_restore_wifi", 1)
```

Comment:
Are you planning to add other states in the future? Otherwise, I think it's better to use true and false here.